packages/core/README.md

```diff
@@ -0,0 +1 @@
+# Core
```

Comment:
You can remove this file as it doesn't add that much value

Reply:
I'll fill it out in a different PR where I'll go around all docs.